site/docs/plugins/session.md

```diff
@@ -326,6 +326,46 @@ If you must use the option (which is of course still possible), you should know
 Make sure you understand the consequences of this configuration by reading [this](../guide/deployment-types.md) article and especially [this](./runner.md#sequential-processing-where-necessary) one.
 :::
 
+### Chat Migrations
+
+If you are using sessions for groups, you should be aware that Telegram migrates regular groups to supergroups under certain circumstances (e.g. [here](https://github.com/telegramdesktop/tdesktop/issues/5593)).
```

Comment:
How is this Telegram Desktop issue related?

Reply:
It's just the only example / documentation for the impact of the migration I could find online. It's just an example of how to distinguish supergroups from normal groups (the part about "Actual behaviour"). Nothing in particular about Telegram Desktop.

Reply:
Still a little misleading, don’t you think?

Reply:
Well, I don't know. Tbh I did not even notice that this is about Telegram Desktop. We could also use [this link](https://stackoverflow.com/questions/54555821/why-telegram-group-automatically-becomes-super-group) instead

Reply:
Can you remove the link?

Reply:
It's documented at https://core.telegram.org/bots/api#message, too. Given how this is a main document that grammY is based on, it could make sense to change the link to that. Given how we constantly repeat information that is found on this page without naming it again, we might just as well remove the link. You decide :)

Reply:
Okay, then I'll just remove the link and leave it to the reader :P 
Don't have access to my computer right now, will update it at the end of the week 👍

Reply:
@Madnex I think you can just rest in your weekend. Thanks for the suggestion, @KnorpelSenf.

Reply:
Awesome, thanks Roj. I can offer to review.